test/postman/nr-starting-app_env.postman_environment.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
This file also looks like exactly the same as `nr-starting-api_test_collection.postman_collection.json`. Is that correct?

Reply:
I'll further simplify those filenames, but they're supposed to be the same within the same pattern, I'll turn them into something like `starting-api.postman_collection.json` and `starting-api.postman_environment.json`